cpp/src/link_analysis/pagerank_1D.cuh

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Author: Alex Fender afender@nvidia.com
+
+#pragma once
+
+#include <rmm/thrust_rmm_allocator.h>
+#include <numeric>
+#include <raft/handle.hpp>
+
+#include "utilities/error_utils.h"
+#include "utilities/spmv_1D.cuh"
+
+namespace cugraph {
+namespace opg {
+
+template <typename VT, typename ET, typename WT>
+class Pagerank {
+ private:
+  size_t v_glob;                     // global number of vertices

```

Comment:
Is there a reason to prefer size_t here?  With the object templated, I believe the following would work and be a little cleaner:
```
  VT v_glob;
  VT v_loc;
  ET e_loc;
```

Reply:
```
VT v_glob{};
VT v_loc{};
ET e_loc{};
```
will be even better.

Reply:
2d430fc